Plugins/Wox.Plugin.Folder/Main.cs

```diff
@@ -102,7 +102,7 @@ private Result CreateFolderResult(string title, string subtitle, string path, Qu
                     {
                         try
                         {
-                            Process.Start(path);
+                            Process.Start("explorer", path);
```

Comment:
nit:  I would prefer a named constant here, as "explorer" is used mutliple times.  It also can give me context that explorer is the name of the process.

Reply:
This is to open a certain path in the file explorer. If you just pass the path it throws as exception on .net core, that is why Im specifying which app should open and the file path as the argument.
But Ill create a variable for the program name. :)